CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ endforeach()
 
 project(ClickHouse)
 
+option(ENABLE_COTIRE "Enable cotire unity build" ON)
```

Comment:
Should we check that submodule exists?

Reply:
dunno... is this what done with the rest of submodules? if so, I will check how it was done with other submodules

Reply:
Yes, we still have limited support for build without submodules.